canvas_modules/common-canvas/src/common-properties/panels/control/control.scss

```diff
@@ -25,3 +24,11 @@
 		pointer-events: none;
 	}
 }
+
+.properties-modal-children .properties-control-panel:first-child {
```

Comment:
Is this needed?  I think since you took out the padding above this should be 0 already.
You should also be able to remove the styling from panels.scss

Reply:
Thats a good point, I removed the unnecessary rules